docker/dockerfile.tri

```diff
@@ -171,7 +185,7 @@ ENV LD_LIBRARY_PATH=/usr/local/hugectr/lib:$LD_LIBRARY_PATH \
     LIBRARY_PATH=/usr/local/hugectr/lib:$LIBRARY_PATH \
     PYTHONPATH=/usr/local/hugectr/lib:$PYTHONPATH
 
-RUN ln -s /usr/local/hugectr/backends/hugectr /opt/tritonserver/backends/
+#RUN ln -s /usr/local/hugectr/backends/hugectr /opt/tritonserver/backends/
```

Comment:
Do we not need this anymore? If so I think we should just remove rather than comment out